clients/drcachesim/tools/filter/record_filter.cpp

```diff
@@ -95,7 +95,7 @@ parse_string(const std::string &s, char sep = ',')
     std::vector<T> vec;
```

Comment:
Do we have any record filter unit test that works with syscalls? If not, let's add one.

Reply:
Done.
I added `code_api|tool.record_filter_keep_func_ids`, which runs `func_id_filter` (part of `record_filter`) with a large function ID and then checks that there is no function marker left in the filtered trace with `basic_counts`.